src/DIRAC/Core/Security/m2crypto/X509Chain.py

```diff
@@ -473,7 +475,9 @@ def generateProxyToString(self, lifetime, diracGroup=False, strength=1024, limit
         return S_OK(proxyString)
 
     # pylint: disable=unused-argument
-    def generateProxyToFile(self, filePath, lifetime, diracGroup=False, strength=1024, limited=False, rfc=True):
+    def generateProxyToFile(
+        self, filePath, lifetime, diracGroup=False, strength=DEFAULT_PROXY_STRENGTH, limited=False, rfc=True
```

Comment:
`rfc` should always be true (since a while) so do we need this parameter at all?